### PR TITLE
support routing capability

### PR DESCRIPTION
**Pull request by meik99**
added support for the DTO routing capability to DTO helm charts

### CODE REVIEW

**Review by DTMad — APPROVED** [review]

**Comment by DTMad** [comment]
Please adapt the msgrouting to routing - We decided yesterday to keep it consistent and go with 'routing' only - see PR for operator: https://github.com/Dynatrace/dynatrace-operator/pull/103

**Comment by DTMad** [comment]
Please also include the PSP for routing once the PR is merged:
https://github.com/Dynatrace/dynatrace-operator/pull/104

**Review by DTMad — APPROVED** [review]
Well done!

File: dynatrace-operator/templates/Common/Routing/serviceaccount-routing.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{{- $platformIsSet := printf "%s" (required "Platform needs to be set to kubernetes or openshift" (include "dynatrace-operator.platformSet" .))}}
```
Comment (lrgar) [inline dynatrace-operator/templates/Common/Routing/serviceaccount-routing.yaml:1]: The roles, etc. are missing this platform check.

Comment (DTMad) [reply dynatrace-operator/templates/Common/Routing/serviceaccount-routing.yaml:15]: Added

**Review by lrgar — APPROVED** [review]

**Pull request merged**